msteams-platform/feedback.md

```diff
@@ -4,28 +4,24 @@ description: The Microsoft Teams product team responds to the developer communit
+|        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repo for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.                                                                                                                                                                                                                       |
+|            **Bot & Messaging Extension SDK issues/suggestions**             |       Submit issues and feature requests to the SDK repo for your bot's language ([C#](https://github.com/Microsoft/botbuilder-dotnet/), [Javascript](https://github.com/Microsoft/botbuilder-js), or [Python](https://github.com/Microsoft/botbuilder-python)). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframewor%20microsoft-teams) using the `botframework & microsoft-teams` tag.                                                                                            |
+| **Samples** |             Submit issues with samples to respective [Microsoft Teams Samples](/microsoftteams/platform/tutorials/code-samples) repo.                                                                                                                                                                                            |
+| **Features suggestions**             |      Please make your features suggestions on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
+|        **Documentation issues**        |                                                                                                                                                                      Click `This page` option in the **Submit and view feedback for** section, available at bottom of respective documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repo.                                                                                                                                                                      |
```

Comment:
Don't use the term "click" See https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click

Reply:
Updated the text as per the style-guide.